usagov/vote-gov
# vote-111 ->  Haitian Creole Accessibility Page

link to ticket: [Vote-111](https://cm-jira.usa.gov/browse/VOTE-111)

to test:

1. pull down branch
2. run `npm run start`
3. visit local site and change to Haitian Creole and select the accessibility page and ensure that page loads and not regression
